WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -509,4 +541,27 @@ private async Task SendTransactionAsync(SmartTransaction transaction)
 
 		return transaction;
 	}
+
+	private void AddToUndoHistory()
+	{
+		_undoHistory.Push((_transaction!, CloneTransactionInfo()));
+		CanUndo = true;
+	}
+
+	private TransactionInfo CloneTransactionInfo()
+	{
+		return new TransactionInfo(_wallet.KeyManager.MinAnonScoreTarget)
+		{
+			Amount = _info.Amount,
+			ChangelessCoins = _info.ChangelessCoins,
+			Coins = _info.Coins,
+			ConfirmationTimeSpan = _info.ConfirmationTimeSpan,
+			FeeRate = _info.FeeRate,
+			IsCustomFeeUsed = _info.IsCustomFeeUsed,
+			MaximumPossibleFeeRate = _info.MaximumPossibleFeeRate,
+			PayJoinClient = _info.PayJoinClient,
+			SubtractFee = _info.SubtractFee,
+			UserLabels = _info.UserLabels
+		};
+	}
```

Comment:
This is dangerous here. If someone added a new property to `TransactionInfo` probably wouldn't know about this clone function and the new property would miss here.

Can you move it inside of the `TransactionInfo` as a `Clone()` method?

Reply:
@soosr fixed